FirebaseSessions/Tests/Unit/ApplicationInfoTests.swift

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+
+@testable import FirebaseSessions
+
+class ApplicationInfoTests: XCTestCase {
+  var appInfo: ApplicationInfo!
+  var mockNetworkInfo: MockNetworkInfo!
+
+  override func setUp() {
+    super.setUp()
+    mockNetworkInfo = MockNetworkInfo()
+    appInfo = ApplicationInfo(appID: "testAppID", networkInfo: mockNetworkInfo)
+  }
+
+  func test_mccMNC_validatesCorrectly() {
+    mockNetworkInfo.mobileCountryCode = "310"
+    mockNetworkInfo.mobileNetworkCode = "004"
+    XCTAssertEqual(appInfo.mccMNC, "310004")
```

Comment:
MNC can also be 2 digits right (judging from the logic in `FIRSESValidateMccMnc`)? Should we add another assert for 2 digit mnc in `test_mccMNC_validatesCorrectly()`?

Reply:
Good call - added a bunch more